python/lib/dependabot/python/file_updater/poetry_file_updater.rb

```diff
@@ -7,7 +7,6 @@
 require "dependabot/python/language_version_manager"
 require "dependabot/python/version"
 require "dependabot/python/requirement"
-require "dependabot/python/python_versions"
```

Comment:
These `require`'s are probably dead on `main` leftover from when `_pavera` did that super useful refactoring to DRY the python version selection logic into a single class.